dspace-spring-rest/src/main/java/org/dspace/app/rest/model/SubmissionFormRowRest.java

```diff
@@ -0,0 +1,30 @@
+/**
+ * The contents of this file are subject to the license and copyright
+ * detailed in the LICENSE and NOTICE files at the root of the source
+ * tree and available online at
+ *
+ * http://www.dspace.org/license/
+ */
+
+package org.dspace.app.rest.model;
+
+import java.util.List;
+
+/**
+ * The InputFormRow REST Resource. It is not addressable directly, only used
+ * as inline object in the InputForm resource
```

Comment:
Javadocs could be improved.  This is a `SubmissionFormRow` and not an `InputFormRow`. Javadocs on the public methods below would be nice too.